src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2000,7 +1953,7 @@ FullWidthRepeat2:
             Dim YearIsTwoDigits As Boolean = False
             Dim DaysToMonth As Integer() = Nothing
             Dim yearIsFirst As Boolean = False
-
+            Dim ch, cx As Char, sep1 As Char
```

Comment:
@TyOverby 
`ch` or `c` is commonly used throughout the scanner code to mean **this character**.
`cx` I'm using to mean some other character. It is typical used where a change to the value of `ch` would affects it meaning. Typically in the `ElseIf` condition.

`sep1` is being renamed to `FirstDateSeparator`